addons/jobs/jobs-service/src/main/java/org/kie/kogito/jobs/service/scheduler/BaseTimerJobScheduler.java

```diff
@@ -40,52 +44,154 @@
  */
 public abstract class BaseTimerJobScheduler implements ReactiveJobScheduler<ScheduledJob> {
 
-    private Logger logger = LoggerFactory.getLogger(BaseTimerJobScheduler.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(BaseTimerJobScheduler.class);
+    public static final long BACKOFF_RETRY_MILLIS = TimeUnit.SECONDS.toMillis(1);
+    public static final long MAX_INTERVAL_LIMIT_TO_RETRY_MILLIS = TimeUnit.SECONDS.toMillis(60);
```

Comment:
Shouldn’t this be configurable via application.properties

Reply:
Yes, I agree, better to have it configurable.